pkg/acsengine/engine_test.go

```diff
@@ -198,19 +198,23 @@ func addTestCertificateProfile(api *api.CertificateProfile) {
 
 func TestVersionOrdinal(t *testing.T) {
 	RegisterTestingT(t)
+	v171 := api.OrchestratorVersion("1.7.1")
 	v166 := api.OrchestratorVersion("1.6.6")
 	v162 := api.OrchestratorVersion("1.6.2")
 	v160 := api.OrchestratorVersion("1.6.0")
 	v153 := api.OrchestratorVersion("1.5.3")
 	v16 := api.OrchestratorVersion("1.6")
 
+	Expect(v171 > v166).To(BeTrue())
```

Comment:
On second glance, I'm not sure what these tests actually accomplish. Not a blocker for this PR.